assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1661,11 +704,11 @@ def apply_annotation(
         api_version = "2023-07-01-preview"
         is_test_connection = True
         token_manager = None
+        model_type = GPT_4
```

Comment:
setting up the workspace connection in AzureML to openai service is (1) difficult, especially for the test workspace used in azureml-assets which we can't control (2) costly to call openai for many test runs - even on our internal RAI dev endpoints we constantly run into quota issues (3) can be flaky and easily lead to test failures depending on quota, networking, etc
For this reason in many cases we mock the results from openai.  However, for dpv2 components which are run remotely I'm not sure of a way to separate out the mock code into the test, since the e2e test code is running remotely.